example/src/tests/createdAtTests.ts

```diff
@@ -83,14 +83,14 @@ test('group createdAt matches listAll', async () => {
   const first = 1
   const second = 0
   assert(
-    alixGroups[first].topic === alixGroup.id,
+    (alixGroups[1] as Group).id === alixGroup.id,
```

Comment:
You could also change this to 🤔 
```suggestion
    alixGroups[1].topic === alixGroup.topic,
```

Reply:
good point, added here 00215afbf787cbc18ae6b9b1c5d0ca2ba7fb126d